### PR TITLE
Flame is added

### PR DESCRIPTION
**Pull request by nachiketsalvi**
Flame is shown up when we click on thrust button and get hidden when we release button.
flame-crashlander

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -14,22 +14,36 @@ var fuelmaxFuel;
 function initGame() {
     lander = new Lander('ship');
     lander.scaleTo(.5);
+    //flame = new Flame('flame');
```
Comment (tcroft) [inline game.js]: Please remove unused code

File: game.js [hunk 2]
```diff
@@ -42,22 +56,36 @@ function loop() {
...
         shipPosition = 0;
     }
-    layout();
+    if(shipSpeed >0 &&
+        shipPosition > 0)
+        {
+            layout();
+        }
+        else{
+            document.getElementById('fuel').innerHTML = 'UFO Crashed';
```
Comment (tcroft) [inline game.js:73]: Note that this shouldn't be in the flame feature branch, it's also not quite the right logic for deciding if it crashed.  You can keep it for now, and fix it when you work on the crash feature

File: game.js [hunk 3]
```diff
@@ -42,22 +56,36 @@ function loop() {
...
     shipSpeed = 0;
     fuel=maxFuel;    
+    
     layout();
+
 }
 
 function startThrust() {
     thrust = true;  
+    initFlame();
```
Comment (tcroft) [inline game.js]: It feels like initFlame() should be in the init() method and then have another showflame() and hideFlame() method to use here

**Comment by nachiketsalvi** [comment]
Removed un-used code.The separate functions created for showing and hiding flame.

**Pull request merged**